client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/MethodReplacementUtils.java

```diff
@@ -0,0 +1,10 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement;
+
+import org.evomaster.client.java.instrumentation.staticstate.MethodReplacementPreserveSemantics;
+
+public class MethodReplacementUtils {
+
+    public static boolean needToPreserverSemantics() {
```

Comment:
why creating this new class just for this method? 
if want to avoid having field directly accessible (ie OO encapsulation), can make the field private in `MethodReplacementPreserveSemantics`, and have getter and setter there